config/db_schemas/user.schema.ts

```diff
@@ -6,6 +6,7 @@ export interface IUser extends TimestampedModel {
   displayName: string;
   email: string;
   hashedPassword: string;
+  profilePicture: string;
```

Comment:
```suggestion
  profilePicture?: string;
```

This should be optional